.github/workflows/publish_image.yml

```
@@ -0,0 +1,34 @@
+name: Publish Docker image
+
+on:
+  push:
+    branches:
+    - master
+    - dev
+
+jobs:
+  push_to_docker_hub:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2.3.4
```

```suggestion
        uses: actions/checkout@v2
```

The `checkout` action maintains a tag for each major release, which is updated on the creation of each new minor or patch release.

https://github.com/actions/checkout/tags

What do you think about just pinning to `v2`? That way, no new work would be required in this repo if they publish a v2.3.5 with some bugfixes.

---

Ah, I didn't notice that! Thanks a lot for the suggestion! It turned out that the same is true for `docker/***` actions, btw.